www/astro.config.mjs

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
   site: `https://beta.create.t3.gg`,
   markdown: {
     shikiConfig: {
-      theme: "material-palenight",
+      theme: "nord",
```

Comment:
subject to change, the `material-palenight` didn't look very good once some actual code was used 🥲 
![image](https://user-images.githubusercontent.com/51714798/193931896-b42527d6-4607-4c9d-8bf8-ce8d22d43a59.png)